windows/src/developer/kmdecomp/savekeyboard.cpp

```diff
@@ -199,13 +204,14 @@ PWCHAR ExtString(PWCHAR str)
 
 	bufpointer = !bufpointer;
 	p = buf[bufpointer]; *p = 0;
+  PWCHAR q = p;
```

Comment:
Some of these look like changes from #5970

Reply:
Yes, apologies, I merged that in early. You can exclude commits 2de8c145c076b361494599fa1fd831809350c7c2 and eb2892ec01091b5c3e77b11397c170ec42ede098 from your review.